udata_front/theme/gouvfr/templates/organization/display.html

```diff
@@ -105,7 +105,7 @@ <h2>{{ _('Datasets') }}<sup>{{ total_datasets }}</sup></h2>
             href="{{ url_for('datasets.list', organization=org.id) }}"
             class="fr-btn fr-btn--secondary fr-icon-eye-line fr-btn--icon-left"
         >
-            {{ _('See %(count)s datasets', count=total_datasets ) }}
+            {{ ngettext('See {0} dataset', 'See {0} datasets', total_datasets) }}
```

Comment:
From what I understand from ngettext, we should use `%(num)d`: https://pythonhosted.org/Flask-BabelEx/#flask.ext.babelex.Domain.ngettext